### ISSUE TITLE
Spending proposal: donate Windows build machine to Isuru

### ISSUE THREAD

**Issue by wolfv** [conda-forge/conda-forge.github.io#1150]
This is a spending proposal to allocate $1500 for a Windows build machine to donate to Isuru to help conda-forge for long running builds on Windows. My proposal is for conda-forge to buy the hardware from our available funds, and donate it to Isuru, in full trust that the hardware will be put to good use.
There are no requirements for this machine to be available for other conda-forge members, but it would be appreciated if the hardware is returned to conda-forge if Isuru decides to cease contributing to conda-forge. After the three year cut off date, the machine is written off.

**Comment by ericdill** [comment]
Not really related to this proposal, but does it make sense to keep a record of these things somewhere? Maybe in our conda-forge google drive somewhere?

This falls under the "Spending of funds", requiring a 50% majority to pass. Given we have 22 members of core (according to core.csv) this means we need 12 votes to pass.

**Comment by scopatz** [comment]
When does the vote end? In a week?

**Comment by wolfv** [comment]
sorry for missing the formalities, thanks both!

Yes 1 week from now sounds good, voting  will end on 8th of September 6 PM GMT+2

As @ericdill said, 50% majority required.

**Comment by CJ-Wright** [comment]
@conda-forge/core

**Comment by msarahan** [comment]
I have a machine that I can donate. Conda forge would need to pay for
shipping and a windows license. Maybe a bigger, newer SSD to speed things
up.

The box I have:
HP z820
2x xeon e5-2687w v2 (16 total cores, 32 threads)
256 GB ddr3
GTX 1070
2 TB spinning disk
2x 250 GB 2015-era SSDs

This said, $1500 would buy a much faster machine these days, though with
less RAM. It does require windows 10 pro because of the dual cpu sockets.

**Comment by msarahan** [comment]
I'm not sure why we have the notion of "writing off" here - that's a concept that applies for tax purposes. We don't deal with taxes. Is it to say that after 3 years, conda-forge will no longer "own" the machine, and whoever has it at that time is free to repurpose it?

While I trust that Isuru would absolutely use this computer in meaningful ways, I think that language around this being a dedicated community resource would be helpful. Perhaps we should name a primary custodian (Isuru) and ask that the primary custodian coordinate remote access for anyone else who might need the machine.

These are things that I think we implicitly agree on, but explicit is better than implicit, right?

**Comment by beckermr** [comment]
I agree @msarahan.

**Comment by wolfv** [comment]
@msaharan yes, your interpretation is what I wanted to say.
I think as a community we have no means to enforce what happens to that hardware anyways, and I think 3 years is a good timeframe for a computer. If it's "returned" (where to, actually?) after that it's just going to rot away in someone else's basement.

Also regarding remote access I think it wouldn't be a great idea to have any expectations. It is a hassle to set up and maintain. Additionally if this computer lives inside someone's home it has access to the network, private files etc. which isn't great.
At QuantStack we have a fast computer that we intended as a build machine and it's constantly down. If we want something that's constantly running we should buy that as a service from elsewhere (i.e. a datacenter).
I think people should be able to nicely ask for remote access but not really expect it to be granted or for it to be available 24/7. That would be up to Isuru to decide.

IMO the easiest setup is to just "give" the hardware without any constraints (just as companies "give" conda-forge money without any constraints :) Companies trust us to do the right thing with their donation and I think we can trust @isuruf to do the right thing with conda-forge's donation. 

But this is my opinion. If this proposal doesn't get enough votes I am happy to support another proposal that includes your language.

Also I think your hardware looks very powerful! IMO it's up to Isuru to decide if he wants to go with your offer or buy something from scratch.

**Comment by mariusvniekerk** [comment]
Windows 10 pro licenses are pretty cheap

**Comment by wolfv** [comment]
today is the last day to vote! Would be great if whoever hasn't voted could give their thumbs-up or thumbs-down 👍 👎 

@beckermr @msarahan if you are not happy with the language, do you want to copy-paste this proposal and make a new vote with your points added? I am happy to vote in favour of yours if that has more potential to reach a majority. 

I just wanted to propose something with minimal overhead for everyone involved as we're all short on time, and this idea with a three-year no strings attached donation seems like it will be the simplest solution (as opposed to also have to maintain some computer that can be accessed at any time by core-members).

**Comment by beckermr** [comment]
Thank you for the note @wolfv. This vote has reached quorum (11/22, kkrause14 is not eligible to vote iiuic) and has thus passed. Therefore I do not want to make a new vote. Nor would I change the language on this one since voting is timeconsuming anyways. While I do think the language should be improved, I also think we are all in general agreement on the intent of the proposal. I hope we can take more care with the language/wording in the future.

**Comment by wolfv** [comment]
If @ericdill is correct, then it should only pass with 12 votes. Just in case.

**Comment by beckermr** [comment]
Ack right. Good catch. I expect it will pass anyway, so I will hold off. If it does not pass, I am happy to make a new one with improved language.

**Comment by beckermr** [comment]
Just for the record, I have voted no for the reasons stated above.

**Comment by ericdill** [comment]
Shall we clarify intent / language at the core meeting on wednesday? Things that I'd specifically like to clarify:

1. expectations around usage of machine
2. expectations around making machine available to other members of conda-forge
3. expectations around returning the machine

I agree with @wolfv's comment above
> IMO the easiest setup is to just "give" the hardware without any constraints (just as companies "give" conda-forge money without any constraints :) Companies trust us to do the right thing with their donation and I think we can trust @isuruf to do the right thing with conda-forge's donation.

In my opinion, the expectations should be the following:

1. We are buying a machine for Isuru to use for painful builds of windows things. There are no expectations around usage beyond that. Isuru can do whatever he wants with this machine, as far as I'm concerned.
2. There is no expectation that Isuru will make this machine available for other members of conda-forge. If he chooses to do so, that is his choice. If other members of conda forge need a machine for windows builds, we can do a similar vote as this one to procure a machine for them.
3. There are no expectations that Isuru will return this machine at any point. As far as I'm concerned this is his machine to do whatever he wishes. If Isuru wishes to pass this machine along to another member of conda-forge, then that is his choice to do so.

In short, we are buying a machine and giving it to Isuru. It is his and he can do whatever he wishes with it.

Would love to hear others opinions on this perspective.

**Comment by wolfv** [comment]
great, it looks like this proposal was accepted 14 YES and 1 NO.

**Comment by mbargull** [comment]
> Shall we clarify intent / language at the core meeting on wednesday?

Yes, please. To me it is not clear what terms, expectations, prerequisites are intended.
(But I'd also like Isuru to have a Windows machine as soon as possible and largely agree with what Eric wrote.)
